src/main/resources/application.properties

```diff
@@ -70,12 +70,17 @@ spring.servlet.multipart.max-request-size=1MB
 spring.servlet.multipart.max-file-size=1MB
 spring.servlet.multipart.file-size-threshold=0
 
+spring.security.oauth2.client.provider.keycloak.realm-uri=http://localhost:8081/realms/demo-app-realm
+spring.security.oauth2.client.provider.keycloak.authorization-uri=${spring.security.oauth2.client.provider.keycloak.realm-uri}/protocol/openid-connect/auth
+spring.security.oauth2.client.provider.keycloak.end-session-endpoint=${spring.security.oauth2.client.provider.keycloak.realm-uri}/protocol/openid-connect/logout
+spring.security.oauth2.client.provider.keycloak.jwk-set-uri=${spring.security.oauth2.client.provider.keycloak.realm-uri}/protocol/openid-connect/certs
+spring.security.oauth2.client.provider.keycloak.token-uri=${spring.security.oauth2.client.provider.keycloak.realm-uri}/protocol/openid-connect/token
+spring.security.oauth2.client.provider.keycloak.user-info-uri=${spring.security.oauth2.client.provider.keycloak.realm-uri}/protocol/openid-connect/userinfo
 spring.security.oauth2.client.provider.keycloak.user-name-attribute=preferred_username
-spring.security.oauth2.client.provider.keycloak.issuer-uri=http://localhost:8081/realms/demo-app-realm
```

Comment:
We can't use `spring.security.oauth2.client.provider.keycloak.issuer-uri` directly, since spring security will rend request with this uri and get the metadata response when PDA is started up.

The metadata is like authorization-uri, end-session-endpoint, jwk-set-uri, token-uri and .user-info-uri etc.